### ISSUE TITLE
Make prettier-ignore work from previous line  

### ISSUE THREAD

**Issue by tintin10q** [prettier/prettier#15391]
**Prettier 3.0.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAHANgQwE9cBLDAfWwCcIBbMuNTAXkwG0BdAbkwHpecVODBgk4VALQkA5lAhCAOlEhQMOAsTIw05GBHIBXbABN88FpmD4kmAMy2ATLYA0mAEY3gmEsZsKQGFQkUNL+rsYQAO5QuBD4xnC+mABm+LhocK5oUPjYaAAWEDDkPn4BMEEh-pgAvjVcSiDOIBDYoujIoPhUNJEACt0IaMggaZFEw81uVPhgANbCAMr4tHAAMsFwyKnpmSDTswswi9izwdLIFQZ7GfSXVNfNaOe4cACKBkVbSDsZzQBWaAAHosXu9PvBtmk-iAAI4QuB9Gh5Eb4NASKBwRKJJogCr4EikEIAYTotHwqNwuFxzxCrwAgiIgm4DPA+uINpiobtmvkYLRcAB1fIkeBoU5gOCLIaikgAN1FhBGYDQkxAcuuAEkoAlYIswEE2vSdYsYMRvr89tQIBlBTNsCNqIxxHKts1ghkqDAkfhpOTuTDTlRPSM3Pg3HBqc1qMEYIKfDB8sgABwABmaQnhJCEPr9FJ+0L2MHD8eMieQDmaBgyABVw8MCzyQHBaBHjAljGt8CEDL64AAxeTkkTnVGsiAgOpAA)

```sh
--parser babel
```

**Input:**

```jsx
const playlist_to_check_promises = []; // prettier-ignore
const playlists_to_update_in_orm= {a: 3323, b: { id: "string", downloaded: false, snapshot_id: "string" }};

```

**Output:**

```jsx
const playlist_to_check_promises = []; // prettier-ignore
const playlists_to_update_in_orm = {
  a: 3323,
  b: { id: "string", downloaded: false, snapshot_id: "string" },
};

```

**Expected behavior:**

```jsx
const playlist_to_check_promises = []; // prettier-ignore
const playlists_to_update_in_orm= {a: 3323, b: { id: "string", downloaded: false, snapshot_id: "string" }};
```

So basically make it so that  `// prettier-ignore` doesn't have to be on its one line.

**Comment by sosukesuzuki** [comment]
Ignoring is applied to first-line. I think this is right behaviour.

---

**Prettier 3.0.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAHANgQwE9cBLDAfWwCcIBbMuNTAXkwG0BGAGk14CYAugG5MAelE4qcGDBJwqAWhIBzKBCkAdKJCgYcBYmRhpyMCOQCu2ACb54LTMHxJMAZld9XPAEYvgmEmsXDRAMKhIoZRCeawgAdyhcCHxrOCDMADN8XDQ4HjQofGw0AAsIGHJA4NCYcMiQzABfRqEtEC4QCGxZdGRQfCoaOIAFAYQ0ZBBsuKIJju8qfDAAa2kAZXxaOAAZCLhkLJy8kAWl1Zg17CWI5WRai2Pc+juqB460G9w4AEULcv2kIdch0AFZoAAea0+Pz+8AO2WBIAAjrC4MMaMVJvg0AooHA0ml2iBavgSKRIgBhOi0fBY3C4IkfSJfACCMnC3gs8GG8l2ePhRw6JRgtFwAHUSiR4GgrmA4GtxlKSAA3KWESZgNBzEDKh4ASSgqVgazA4W6LMNaxgxABQOO1AguTFi2wk2ojHkyv2HQiuSoMHR+GUNIFiKuVD9k28+G8cAZHWoERgYsCMBKyAAHAAGDpSFEkKSB4O0wEI44wGMp6xp5B8DoWXIAFRjE1LgpAcFosesqWs23wkQsQbgADF1DSZDcsVyICBmkA)

```sh
--parser babel
```

**Input:**

```jsx
const playlist_promises = [1,   2]; // prettier-ignore
const playlists_to_update = {a: 3323, b: { id: "string", downloaded: false, snapshot_id: "string" }};

```

**Output:**

```jsx
const playlist_promises = [1,   2]; // prettier-ignore
const playlists_to_update = {
  a: 3323,
  b: { id: "string", downloaded: false, snapshot_id: "string" },
};

```
